test/unit/models/discordactions.test.js

```diff
@@ -567,4 +569,31 @@ describe("discordactions", function () {
       }
     }).timeout(10000);
   });
+
+  describe("addInviteToInviteModel", function () {
+    let addStub;
+
+    beforeEach(function () {
+      addStub = sinon.stub(discordInvitesModel, "add").resolves({ id: "invite-test-id" });
+    });
+
+    afterEach(function () {
+      addStub.restore();
+    });
+
+    it("should add invite in the invite model for user", async function () {
+      const inviteObject = { userId: "kfjkasdfl", inviteLink: "discord.gg/xyz" };
+      const inviteId = await addInviteToInviteModel(inviteObject);
+      expect(inviteId).to.exist; // eslint-disable-line no-unused-expressions
```

Comment:
why is eslint disabled here?

Reply:
so eslint was giving a error where they were expecting a function here, so that is why disabled it

Reply:
Why not fix the issues? shouldn't we be fixing the root cause, instead of ignoring it

Reply:
okay, let me check what can be done

Reply:
Hey @prakashchoudhary07 I read up on this, this is how we do this assertion in chai, the issue is that eslint is thinking that we are just accessing a particular value and nothing is done with it, that is why it is throwing the error, so I think this is fine to disabled this in this test